Source/WebCore/rendering/style/PositionArea.h

```diff
@@ -47,6 +47,32 @@ enum class PositionAreaAxis : uint8_t {
     Block  = 0b111,
 };
 
+WTF::TextStream& operator<<(WTF::TextStream&, PositionAreaAxis);
+
+// Get the opposite axis of a given axis.
+static inline PositionAreaAxis flipPositionAreaAxis(PositionAreaAxis axis)
```

Comment:
No need to inline this.

Reply:
It's a function declaration in a header file so the `inline` is needed.